app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/Events.kt

```diff
@@ -167,7 +148,7 @@ fun Events(
             horizontalAlignment = Alignment.CenterHorizontally,
             modifier = Modifier.padding(innerPadding)) {
               Spacer(modifier = Modifier.height(5.dp))
-              GoMeetSearchBar(nav, query, NavBarUnselected, Color.DarkGray)
+              GoMeetSearchBar(nav, query, NavBarUnselected, MaterialTheme.colorScheme.tertiary)
```

Comment:
i think it would be better to not change this because it looks bad in dark mode 
![Capture d'écran 2024-05-14 081239](https://github.com/SwEnt-Project-G18/SwEntApp/assets/125990940/7e3d5b8d-0164-46ee-a9db-578385abaf66)

Reply:
i also think it would be better to change the color of the search bar since the current one is green-ish and if we are using blue as accent color it doesn't look very good.

Reply:
This is fixed